gallery/src/components/NxDropdownIconOnly/NxDropdownIconOnlyPage.tsx

```diff
@@ -0,0 +1,169 @@
+import NxDropdownIconOnlyDisabledExample from './NxDropdownIconOnlyDisabledExample';
+
+const nxDropdownIconOnlyNavigationExampleCode = require('./NxDropdownIconOnlyNavigationExample?raw'),
+    nxDropdownIconOnlyLinksExampleCode = require('./NxDropdownIconOnlyLinksExample?raw'),
+    nxDropdownIconOnlyDisabledExampleCode = require('./NxDropdownIconOnlyLinksExample?raw');
+
+const NxDropdownPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        A variation of the dropdown menu component which uses a single icon as its toggle.
+        <NxCode>NxDropdownIconOnly</NxCode> shares all of the same props as{' '}
+        <NxTextLink href="#/pages/NxDropdown" className="nx-dropdown-button">
+          NxDropdown
+        </NxTextLink>{' '}
+        as well as the ones listed below. Like <NxCode>NxDropdown</NxCode> this is for menus of nagivation links or
+        action-triggering buttons.  It is <em>not</em> a form select field. By default the dropdown can display a
+        maximum of 10 items before it scrolls the contents of the dropdown menu.
+      </NxP>
+      <NxTile.Subsection>
+        <NxTile.SubsectionHeader>
+          <NxH3>Props:</NxH3>
+        </NxTile.SubsectionHeader>
+        <NxTable>
+          <NxTable.Head>
+            <NxTable.Row>
+              <NxTable.Cell>Prop</NxTable.Cell>
+              <NxTable.Cell>Type</NxTable.Cell>
+              <NxTable.Cell>Required</NxTable.Cell>
+              <NxTable.Cell>Details</NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Head>
+          <NxTable.Body>
+            <NxTable.Row>
+              <NxTable.Cell>icon</NxTable.Cell>
+              <NxTable.Cell>string | VDOM</NxTable.Cell>
+              <NxTable.Cell>Yes</NxTable.Cell>
+              <NxTable.Cell>
+                The name of the FontAwesome icon to use in the button. The default is <NxCode>faEllipsisV</NxCode>
+                {' '} <NxFontAwesomeIcon icon={faEllipsisV}></NxFontAwesomeIcon>.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell>title</NxTable.Cell>
+              <NxTable.Cell>string | NxTooltip Props</NxTable.Cell>
+              <NxTable.Cell>Yes</NxTable.Cell>
+              <NxTable.Cell>
+                Describes the text for a tooltip to be placed on the dropdowns' toggle element. For accessibility
+                reasons this prop is required on <NxCode>NxDropdownIconOnly</NxCode>. There are two ways to specify the
+                tooltip: the simpler way is to simply specify the tooltip text as a string. If control of more complex
+                tooltip options is desired, an object can be passed which will serve as the props for
+                {' '}<NxCode>NxTooltip</NxCode>.
+              </NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Body>
+        </NxTable>
+      </NxTile.Subsection>
+
+      <NxTile.Subsection>
```

Comment:
This subsection and the next are copied verbatim from the `NxDropdown` page, right? I don't think we should duplicate it, but rather refer back to it there.

Reply:
> This subsection and the next are copied verbatim from the `NxDropdown` page, right? I don't think we should duplicate it, but rather refer back to it there.

https://github.com/sonatype/sonatype-react-shared-components/pull/465/commits/94abc6ae861cbd13147bbbf25d88876f640c708b